flowforge/materials/Fluid.py

```diff
@@ -228,6 +268,14 @@ def viscosity(self, h):
         d = -2.019e-9
         return a + (b*self.temperature(h)) + (c*(self.temperature(h)**2)) + (d*(self.temperature(h)**3))
```

Comment:
So find this mildly amusing, but the INL reference calls out Janz for leaving out `T^3` from the equation, and then proceeds to leave out `T` for the `b` term in its transcription....  Maybe note in the comment that the INL document left out the `T` for the `b` term, just for completeness...

Reply:
![image](https://github.com/user-attachments/assets/f1f1e7cb-58ba-4858-bbd3-19d5b7d36f63)

Reply:
Here it is in the original Janz reference:
![image](https://github.com/user-attachments/assets/dfe421e9-c8f2-4592-846d-6810ee67f585)

Reply:
That is such a great catch and also very ironic. Well I just found out that there is a more recent paper with that revision [2013](https://inldigitallibrary.inl.gov/sites/STI/STI/5698704.pdf)

![image](https://github.com/user-attachments/assets/4498db5a-90f1-4b86-ad43-057671112b94)


Of course right after I just cited everything in the code with the [2010](https://www.osti.gov/biblio/980801) paper.

What do you think I should do? Make note of it or rewrite all of the reference to match the 2013 paper?

Reply:
Use the newer reference

Reply:
Updated and Done